kodiUtilities.py

```diff
@@ -90,71 +90,18 @@ def checkExclusion(fullpath):
-            return True
-    ExcludePath4 = getSetting('ExcludePath4')
-    if ExcludePath4 != "" and getSettingAsBool('ExcludePathOption4'):
-        if fullpath.startswith(ExcludePath4):
-            logger.debug("checkExclusion(): Video is from location, which is currently set as excluded path 4.")
-            return True
-    ExcludePath5 = getSetting('ExcludePath5')
-    if ExcludePath5 != "" and getSettingAsBool('ExcludePathOption5'):
-        if fullpath.startswith(ExcludePath5):
-            logger.debug("checkExclusion(): Video is from location, which is currently set as excluded path 5.")
-            return True
-    ExcludePath6 = getSetting('ExcludePath6')
-    if ExcludePath6 != "" and getSettingAsBool('ExcludePathOption6'):
-        if fullpath.startswith(ExcludePath6):
-            logger.debug("checkExclusion(): Video is from location, which is currently set as excluded path 6.")
-            return True
-    ExcludePath7 = getSetting('ExcludePath7')
-    if ExcludePath7 != "" and getSettingAsBool('ExcludePathOption7'):
-        if fullpath.startswith(ExcludePath7):
-            logger.debug("checkExclusion(): Video is from location, which is currently set as excluded path 7.")
-            return True
-    ExcludePath8 = getSetting('ExcludePath8')
-    if ExcludePath8 != "" and getSettingAsBool('ExcludePathOption8'):
-        if fullpath.startswith(ExcludePath8):
-            logger.debug("checkExclusion(): Video is from location, which is currently set as excluded path 8.")
-            return True
-    ExcludePath9 = getSetting('ExcludePath9')
-    if ExcludePath9 != "" and getSettingAsBool('ExcludePathOption9'):
-        if fullpath.startswith(ExcludePath9):
-            logger.debug("checkExclusion(): Video is from location, which is currently set as excluded path 9.")
-            return True
-    ExcludePath10 = getSetting('ExcludePath10')
-    if ExcludePath10 != "" and getSettingAsBool('ExcludePathOption10'):
-        if fullpath.startswith(ExcludePath10):
-            logger.debug("checkExclusion(): Video is from location, which is currently set as excluded path 10.")
-            return True
-    ExcludePath11 = getSetting('ExcludePath11')
-    if ExcludePath11 != "" and getSettingAsBool('ExcludePathOption11'):
-        if fullpath.startswith(ExcludePath11):
-            logger.debug("checkExclusion(): Video is from location, which is currently set as excluded path 11.")
-            return True
-    ExcludePath12 = getSetting('ExcludePath12')
-    if ExcludePath12 != "" and getSettingAsBool('ExcludePathOption12'):
-        if fullpath.startswith(ExcludePath12):
-            logger.debug("checkExclusion(): Video is from location, which is currently set as excluded path 12.")
-            return True
 
-    return False
+    for x in xrange(2,12):
+        utilities.checkExcludePath(getSetting('ExcludePath%i' % x).encode('utf-8'), getSettingAsBool('ExcludePathOption%i' % x), fullpath, x)
 
+    return False
```

Comment:
I believe this does always return `False`.
I suspect you actually wanted something more like:
```python
found = False
for x in xrange(...):
    found |= utilities.checkExcludePath(...)

return found